proto/rpc.proto

```diff
@@ -28,104 +24,165 @@ message Transaction {
 }
 
 message Block {
-  uint32 version = 1;
-  Hash milestoneBlockHash = 2;
-  Hash prevBlockHash = 3;
-  Hash tipBlockHash = 4;
-  uint32 diffTarget = 5;
-  uint32 nonce = 6;
-  uint64 time = 7;
-  repeated uint32 proof = 8;
-  repeated Transaction transactions = 9;
+  string hash = 1;
+  uint32 version = 2;
+  string msHash = 3;
+  string prevHash = 4;
+  string tipHash = 5;
+  uint32 diffTarget = 6;
+  uint32 nonce = 7;
+  uint64 time = 8;
+  repeated uint32 proof = 9;
+  repeated Transaction transactions = 10;
+}
+
+message Vertex {
+    Block block = 1;
+    uint64 height = 2;
+    bool isMilestone = 3;
+    uint32 redemptionStatus = 4;
+    repeated bool txStatus = 5;
+    uint64 rewards = 6;
```

Comment:
We should also include `snapshot`. Better to add a message type called `milestone`.

Reply:
But what information of `milestone` is important for rpc users? If we are going to provide information about corresponding level set, user can further inquiry for it via other rpc commands.

Reply:
Hash rate, targets, etc.